lib/quandl/config.rb

```diff
@@ -35,7 +35,7 @@ def configurable_attributes
     private
 
     def read_config(file_name)
-      raw_config = File.read(Pathname.new(project_root).join('config', "#{file_name}.yml"))
+      raw_config = File.read(Pathname.new(project_root || '').join('config', "#{file_name}.yml"))
```

Comment:
Perhaps you could move that into the `def project_root` method. Also would it make sense to set the project root in those cases to the `cwd`. ex: `Dir.pwd`